### PR TITLE
Holiday cleanup and bug fixes

### PR DESCRIPTION
**Pull request by pmur**
Resolve the autotester bug when rounding small values uncovered by #112 

Likewise, cleanup trunctdsd2 on ppc and remove the unused sysdep dirs.

Likewise, fix power9 compilation error due to missing semicolon.

### CODE REVIEW

**Comment by pmur** [comment]
@pdrocaldeira would you double check this with your nextAfter fix?

**Comment by pmur** [comment]
@stliibm this promotion of powd64 to shared code might require an ulp's update for z.

File: ieee754r/powd64.c [hunk 1]
```diff
@@ -23,7 +23,201 @@
...
+#ifndef _DECIMAL_SIZE
+#  define _DECIMAL_SIZE 64
+#  include <decimal64.h>
+#endif
+
+#define FUNCTION_NAME pow
+
+#include <dfpmacro.h>
+
+static DEC_TYPE
```
Comment (stliibm) [inline ieee754r/powd64.c]: There is a whitespace after "static DEC_TYPE".

File: ieee754r/powd64.c [hunk 2]
```diff
@@ -23,7 +23,201 @@
...
+		    return DEC_INFINITY;
+		}
+	    }
+	  else
+	    {
+	      if (x > 0.DD)
+		{ /* x == +inf */
+		  if (y < 0.DD)
+		    return 0.DD;
+		  else
```
Comment (stliibm) [inline ieee754r/powd64.c]: There is a whitespace after "else".

**Comment by stliibm** [comment]
@pmur I've build and run the testsuite on s390x (64bit) / s390 (31bit) with / without hw support. All tests are fine and no ulps update is needed.

**Pull request merged**